extras/rootless/containerd-rootless.sh

```diff
@@ -145,17 +145,17 @@ if [ -z $_CONTAINERD_ROOTLESS_CHILD ]; then
 	# * /run:     copy-up is required so that we can create /run/containerd (hardcoded) in our namespace
 	# * /var/lib: copy-up is required so that we can create /var/lib/containerd in our namespace
 	exec rootlesskit \
-		--state-dir=$CONTAINERD_ROOTLESS_ROOTLESSKIT_STATE_DIR \
-		--net=$net --mtu=$mtu \
-		--slirp4netns-sandbox=$CONTAINERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX \
-		--slirp4netns-seccomp=$CONTAINERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SECCOMP \
-		--disable-host-loopback --port-driver=$CONTAINERD_ROOTLESS_ROOTLESSKIT_PORT_DRIVER \
+		--state-dir="$CONTAINERD_ROOTLESS_ROOTLESSKIT_STATE_DIR" \
+		--net="$net" --mtu="$mtu" \
+		--slirp4netns-sandbox="$CONTAINERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX" \
+		--slirp4netns-seccomp="$CONTAINERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SECCOMP" \
+		--disable-host-loopback --port-driver="$CONTAINERD_ROOTLESS_ROOTLESSKIT_PORT_DRIVER" \
 		--copy-up=/etc --copy-up=/run --copy-up=/var/lib \
 		--propagation=rslave \
-		$CONTAINERD_ROOTLESS_ROOTLESSKIT_FLAGS \
-		$0 $@
+		"$CONTAINERD_ROOTLESS_ROOTLESSKIT_FLAGS" \
+		"$0" "$@"
```

Comment:
This "$CONTAINERD_ROOTLESS_ROOTLESSKIT_FLAGS" here is the reason you are failing to start containerd in rootless.
You need to leave that one unquoted so that it does expand.

Reply:
Makes sense. Looks like it contains multiple flags, so it should be expanded.